modules/ding_event/ding_event.pages_default.inc

```diff
@@ -969,6 +975,8 @@ node/%node:nid/events
   $display->cache = array();
   $display->title = '';
   $display->uuid = '242cdbfd-b37b-4e9e-b3b0-415f58651d59';
+  $display->storage_type = 'page_manager';
```

Comment:
How is this related to the case?

Reply:
This is required by panels module in order to fix some access permissions issues, introduced in version 7.x-3.6, when on pane editing administrators had "access denied" error.